assets/data-port/style.scss

```diff
@@ -159,24 +159,6 @@ $alert-red: #d94f4f;
 				display: flex;
 				justify-content: flex-end;
 				align-items: center;
-
-				.components-button {
-					padding: 6px 24px;
-					background: $green;
-					border-color: $green;
-					height: 40px;
```

Comment:
The `padding` and `height` rules are still needed to make this button a bit larger, as the Sensei Color styles only take care of colors.

Reply:
Ahh good catch. Not sure what happened when testing this locally.